eng/Build.ps1

```diff
@@ -339,6 +342,7 @@ try {
         TestUsingNUnit -testProject "$RepoRoot\tests\FSharp.Compiler.UnitTests\FSharp.Compiler.UnitTests.fsproj" -targetFramework $desktopTargetFramework
         TestUsingNUnit -testProject "$RepoRoot\tests\FSharp.Compiler.LanguageServer.UnitTests\FSharp.Compiler.LanguageServer.UnitTests.fsproj" -targetFramework $desktopTargetFramework
         TestUsingNUnit -testProject "$RepoRoot\tests\FSharp.Compiler.Private.Scripting.UnitTests\FSharp.Compiler.Private.Scripting.UnitTests.fsproj" -targetFramework $desktopTargetFramework
+        TestUsingNUnit -testProject "$RepoRoot\tests\FSharp.DependencyManager.Testing\FSharp.DependencyManager.Testing.fsproj" -targetFramework $desktopTargetFramework
```

Comment:
Also add this test project under `if ($testCoreClr)` below.

Reply:
And add to `eng/build.sh` for the Linux/MacOS coverage.

Reply:
@brettfo … Linux coverage … that's harsh :-)